### PR TITLE
nri-mssql-experimental

### PR DESCRIPTION
**Pull request by msummers-nr**
# Summary

An experimental version of the original New Relic nri-mssql with added query plan logging to the Log API. nri-mssql would not accept the change as a PR due to the use of the Log API so this will live in newrelic-experimental on its own.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work? Yes
- [x] Did you check your dashboard image quality? -  Do they look good? N/A
- [x] Did you check that your alerts actually work? N/A
- [x] Did you include an InstallPlan and Documentation reference? Yes
- [x] Did you check your descriptive content for voice, tone, spelling and grammar errors? Yes
- [x] Did you attach images of your dashboards to the PR so we can see them working? N/A

### Screenshots

Not applicable

### CODE REVIEW

File: quickstarts/mssql-queryplan/config.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Name of the quickstart (required)
+name: nri-mssql-experimental
+title: Microsoft SQL Server with Query Plan logging
+description: |
+  The original SQL Server nri with added Query Plan logging
```
Comment (jpvajda) [inline quickstarts/mssql-queryplan/config.yml:5]: Would you be able to add more content to the description? This is a little light on product value and why it's useful to an end user.

File: quickstarts/mssql-queryplan/config.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+summary: |
+  Send SQL Server Query Plans to the New Relic Logging API
```
Comment (jpvajda) [inline quickstarts/mssql-queryplan/config.yml:8]: Same comment as above, this summary could use a little more detail.

File: quickstarts/mssql-queryplan/config.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+# Authors of the quickstart (required)
+authors:
+  - New Relic Labs
+
+documentation:
+  - name: Apigee Distributed Tracing documentation
+    description: |
+      Installation and configuration instructions
+    url: https://github.com/newrelic-experimental/nri-mssql-experimental
```
Comment (jpvajda) [inline quickstarts/mssql-queryplan/config.yml:30]: Is there a reference to a docs.newelic.com page you can use here, along with a link to the repository?

**Review by jpvajda — CHANGES_REQUESTED** [review]

File: quickstarts/mssql-queryplan/config.yml (hunk 3 above)
Comment (msummers-nr) [reply quickstarts/mssql-queryplan/config.yml:30]: @jpvajda We haven't been adding our newrelic-experimental projects to doc.newrelic.com as they're unsupported and not "Product"

**Comment by amit-y** [comment]
👋  @jpvajda this PR can be closed without merging. Please see #793 for an updated version of this quickstart. Thanks!

**Pull request closed without merging**